security/src/main/java/com/fincity/security/service/ClientUrlService.java

```diff
@@ -287,6 +287,11 @@ public Mono<ClientUrl> createForRegistration(ClientUrl entity) {
 				.flatMap(cacheService.evictAllFunction(SSLCertificateService.CACHE_NAME_CERTIFICATE_LAST_UPDATED_AT));
 	}
 
+	public Mono<ClientUrl> getSubDomain(ULong clientId, String appcode){
```

Comment:
Vishwas just finished getclienturl in clientUrlService. Please merge master and use that.